django_project/ford3/models/provider.py

```diff
@@ -262,3 +265,149 @@ def get_location_as_dict(self):
                 'location_value_x': 0,
                 'location_value_y': 0})
         return result
+
+    def import_excel_data(self, data):
```

Comment:
how do we call this method?

Reply:
I feel this method (and the others) should be somewhere else than in the Provider model.